File: Modul-01/python/PRAK109-2310817210009-MuhammadIbnuSina.py
# Input jumlah pasukan yang dibawa Yu Zhong
total_pasukan = float(input("Jumlah pasukan yang dibawa Yu Zhong: "))

# Input jumlah pahlawan
jumlah_pahlawan = float(input("Jumlah pahlawan: "))

# Menghitung jumlah pasukan yang harus dikalahkan setiap pahlawan
pasukan_per_pahlawan = total_pasukan / jumlah_pahlawan

# Menampilkan hasil tanpa titik desimal
print(
    f"Jumlah pasukan yang harus dikalahkan setiap pahlawan adalah {int(pasukan_per_pahlawan)} pasukan")

File: Modul-01/c/PRAK109-2310817210009-MuhammadIbnuSina.c
#include <stdio.h>

int main() {
    float totalPasukan, jumlahPahlawan;

    // Input jumlah pasukan yang dibawa Yu Zhong
    printf("Jumlah pasukan yang dibawa Yu Zhong: ");
    scanf("%f", &totalPasukan);

    // Input jumlah pahlawan
    printf("Jumlah pahlawan: ");
    scanf("%f", &jumlahPahlawan);

    // Menghitung jumlah pasukan yang harus dikalahkan setiap pahlawan
    float pasukanPerPahlawan = totalPasukan / jumlahPahlawan;

    // Menampilkan hasil
    printf("Jumlah pasukan yang harus dikalahkan setiap pahlawan adalah %.0f pasukan\n", pasukanPerPahlawan);

    return 0;
}
